package.json

```diff
@@ -3,7 +3,14 @@
   "description": "XRPL Data Visualization",
   "version": "1.5.0",
   "dependencies": {
+    "@babel/eslint-parser": "^7.14.7",
     "@google-cloud/bigquery": "^5.5.0",
+    "@types/jest": "^26.0.24",
+    "@types/node": "^16.4.3",
+    "@types/react": "^17.0.15",
+    "@types/react-dom": "^17.0.9",
+    "@types/react-i18next": "^8.1.0",
+    "@types/react-router-dom": "^5.1.8",
```

Comment:
none of these are needed at runtime so they could all be "devDependencies". It doesn't really matter though, since we aren't publishing this as a package on npm; the distinction between "dependencies" and "devDependencies" is totally unnecessary in this project.

Reply:
yeah that's why I didn't bother to move them, but I can if you want

Reply:
Just did it anyways